dotcom-rendering/src/web/components/SignInGate/gateDesigns/SignInGateMain.tsx

```diff
@@ -33,7 +33,7 @@ export const SignInGateMain = ({
 			<div css={firstParagraphOverlay} />
 			<h1 css={headingStyles}>You need to register to keep reading</h1>
 			<p css={bodyBold}>
-				It’s still free to read - this is not a paywall
+				It’s still free to read – this is not a paywall
```

Comment:
Is this change neccessary?

Reply:
Just think might be tidier to fix any Main gate  UI issues in a separate PR :)

Reply:
Yeah good shout, it will also stop it conflicting with the other PR we have been working on. I'll revert this change